pages/common/lz4.md

```diff
@@ -0,0 +1,27 @@
+# lz4
+
+> Compress or decompress .lz4 files.
+
+- Compress a file:
+
+`lz4 {{file}}`
+
+- Decompress a file:
+
+`lz4 -d {{file.lz4}}`
+
+- Decompress a file and write to stdout:
+
+`lz4 -dc {{file.lz4}}`
+
+- Package and compress a directory and its contents:
+
+`tar cvf - {{/path/to/dir}} | lz4 - {{dir.tar.lz4}}`
```

Comment:
Could you change the absolute path to `{{path/to/dir}}`?

Reply:
Done!